### PR TITLE
feat(cards): add the ribbon image

### PR DESCRIPTION
**Pull request by GusBaamonde**
- the pages for the cards and for the doppler plus layout were created
- links to library navigation added
- doppler plus and cards component html created
- four classes were created to modify the position of the ribbons
- the hover effect of the cards was created
- "Data-ribbon" is used to edit the contents of the ribbon

https://cdn.fromdoppler.com/doppler-ui-library/build918/cards-component.html

cards-ribbons

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/templates/cards-component.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        >
+          Back
+        </a>
+      </header>
+
+      <div class="dp-gray-page p-t-54 p-b-54">
+        <div class="dp-align-center">
+          <div>
+            <h3 class="m-l-30">Card Ribbon left-top</h3>
+            <div class="dp-card-ribbon-lt" data-ribbon="Recomendado"></div>
```
Comment (cbernat) [inline assets/templates/cards-component.html:76]: What happens if you add more text here? Say like something that occupies two lines.

**Comment by cbernat** [comment]
image

You need to remove one ../ for the image to be located correctly.

File: assets/templates/cards-component.html (hunk 1 above)
Comment (cbernat) [reply assets/templates/cards-component.html:76]: image

This is to think for the future only, that´s all.

Comment (cbernat) [reply assets/templates/cards-component.html:76]: http://cdn.fromdoppler.com/doppler-ui-library/build917/cards-component.html

Comment (GusBaamonde) [reply assets/templates/cards-component.html:76]: This does not work like that, there is no space for two words, the design is not prepared for this. It is not an elastic ribbon.
Surely if they want to add more words they will change the design.
This accepts 1 word or 2 very small

Comment (GusBaamonde) [reply assets/templates/cards-component.html:76]: https://cdn.fromdoppler.com/doppler-ui-library/build918/cards-component.html

Comment (cbernat) [reply assets/templates/cards-component.html:76]: And in the case we need to support more than one size of ribbon, is it prepared? Or have you thought about this?

Comment (GusBaamonde) [reply assets/templates/cards-component.html:76]: In all the designs they have there are no variants of ribbons but it is something that can be done without difficulty 🐇

Comment (mdirago) [reply assets/templates/cards-component.html:76]: Great job Gus! About the ribbons as Gus has said, there aren´t another iterations. Only it´s designed to contain one word, or two short words. If another requerimient shows up we will consider it! :)

**Pull request merged**